### DISCUSSION TITLE
Add support to reorder views

### DISCUSSION THREAD

**Discussion by diegobernardes** [community/community#6345] [Projects and Issues] [answered]
I have some views created for a project but I can't reorder them.

**Comment by tklein1801** [comment, marked as answer]
Duplicate of #4803

**Comment by azenMatt** [comment]
Hi @diegobernardes @tklein1801  - we just released re-orderable view tabs as part of our [recent updates](https://github.blog/changelog/2021-11-18-the-new-github-issues-11-18-update/). 

You can now drag and drop the views into any order you want 😄

**Reply by tklein1801** [reply]
Awesome, thank you

**Reply by diegobernardes** [reply]
Thanks 🙇🏻‍♂️
